launch/launch/invalid_launch_file_error.py

```diff
@@ -27,9 +27,17 @@ def __init__(self, extension='', *, likely_errors=None):
                 'The launch file may have a syntax error, or its format is unknown'
             )
         else:
-            self._error_message = (
-                'Caught exception when trying to load file of format [{}]: {}'
-            ).format(self._extension, self._likely_errors[0])
+            if len(self._likely_errors) == 1:
+                self._error_message = (
+                    'Caught exception when trying to load file of format [{}]: {}'
+                ).format(self._extension, self._likely_errors[0])
+            else:
+                self._error_message = (
+                    'Caught multiple exceptions when trying to load file of format [{}]:'
+                ).format(self._extension)
+                for error in self._likely_errors:
+                    self._error_message += '\n - {}'.format(error)
+
```

Comment:
Personally I think this would be better if it were consistent between the single and multiple exception cases.  That is, I think the following would do here:

```suggestion
            self._error_message = (
                'Caught {} when trying to load file of format [{}]:'
            ).format('multiple exceptions' if len(self._likely_errors) > 1 else 'exception',
                      self._extension)
            for error in self._likely_errors:
                self._error_message += '\n - {}'.format(error)

```